.codecov.yml

```diff
@@ -8,9 +8,4 @@ coverage:
     patch:
       default:
         informational: true
-comment:
-  require_head: no
-  require_base: no
-  require_changes: false
-  layout: "reach,diff,files"
-  behavior: default
```

Comment:
You finally had enough of it then?!

Reply:
The comment's inconsistency annoys me.. cant figure out why it sometimes updates and sometimes doesnt 😆